pkg/machines/components/vcpuModal.css

```diff
@@ -0,0 +1,17 @@
+.vcpu-modal-grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 1fr));
+  grid-row-gap: 1em;
+}
+
+.vcpu-modal-grid input[type=number] {
+    max-width: 7rem;
+}
+
+.vcpu-modal-grid .dropdown {
+    min-width: 11rem;
```

Comment:
Used min-width to bump up the minimum width, as the width is forced to auto elsewhere (in the grid layout). As we really care about a minimum with here, it's more correct.